src/network/networkd-ipv6ll.c

```diff
@@ -74,6 +74,48 @@ bool link_may_have_ipv6ll(Link *link) {
         return false;
 }
 
+IPv6LinkLocalAddressGenMode link_get_ipv6ll_addrgen_mode(Link *link) {
+        if (!link_ipv6ll_enabled(link))
+                return IPV6_LINK_LOCAL_ADDRESSS_GEN_MODE_NONE;
+
+        if (link->network->ipv6ll_address_gen_mode >= 0)
```

Comment:
assert/check link and link->network?

Reply:
`link->network` is handled `link_ipv6ll_enabled()`. Added `assert(link);`.